### PR TITLE
last of numpy edits

### PR DESCRIPTION
**Pull request by Kully**
Re: https://github.com/plotly/documentation/issues/604

@cldougl ready to review

### CODE REVIEW

File: _posts/numpy/reshape/2015-06-30-Reshape.html [hunk 1]
```diff
@@ -178,4 +178,124 @@ <h4 id="Reshape-an-Array">Reshape an Array<a class="anchor-link" href="#Reshape-
...
+        Fortran- contiguous) of the returned array.
+    
+    See Also
+    --------
+    ndarray.reshape : Equivalent method.
+    
+    Notes
+    -----
+    It is not always possible to change the shape of an array without
+    copying the data. If you want an error to be raise if the data is copied,
```
Comment (cldougl) [inline _posts/numpy/reshape/2015-06-30-Reshape.html]: raised

**Comment by cldougl** [comment]
thanks for doing this @Kully ! 💃 after raise/raised typo!

File: _posts/numpy/reshape/2015-06-30-Reshape.html (hunk 1 above)
Comment (Kully) [inline _posts/numpy/reshape/2015-06-30-Reshape.html]: actually, that's the text from the `help(reshape)`, so it's actually a `Numpy` issue

Comment (Kully) [inline _posts/numpy/reshape/2015-06-30-Reshape.html]: let me try rerunning and see if it looks better

Comment (cldougl) [inline _posts/numpy/reshape/2015-06-30-Reshape.html]: ok if not - no worries, I figured it was a docstring issue, but I thought maybe one of ours.

Comment (Kully) [inline _posts/numpy/reshape/2015-06-30-Reshape.html]: The typo is still there. Can I merge?

**Pull request merged**